### PR TITLE
chore: Updating content in components

### PR DESCRIPTION
**Pull request by bsonefeld**
Closes [#380 ](https://github.ibm.com/Bluemix/carbon-issues/issues/380)(internal).

Updated content in components to be consistent across Design Kit, Carbon website, and Carbon Components. 

## Changelog

**Changed**
- Content in all components. 

**Removed**

- Sentence casing from labels.

### CODE REVIEW

File: .storybook/components/ButtonStory.js [hunk 1]
```diff
@@ -173,11 +173,11 @@ storiesOf('Buttons', module)
     () =>
       <div>
         <Button kind="danger" {...buttonEvents} className="some-class">
-          Danger Button
+          Danger nutton
```
Comment (joshblack) [inline .storybook/components/ButtonStory.js]:
```diff
- Danger nutton
+ Danger button
```

I think?

Comment (tw15egan) [reply .storybook/components/ButtonStory.js]: I aint done nutton wrong!

**Comment by marijohannessen** [comment]
Hey @bsonefeld , looks like there's a bunch of merge conflicts happening - I can help you fix them tomorrow 😄

**Comment by bsonefeld** [comment]
Merge conflicts resolved thanks to @marijohannessen , and @joshblack I addressed your changes, could I get another review?

**Comment by marijohannessen** [comment]
Looks like the tests are failing now. Could you go to the file `Form-story.js` and remove the lines 90-93?
```
const fieldsetTextInputProps = {
   className: 'some-class',
    legendText: 'Text Input label',
};
```
This part ^^. Let me know if you need any help!

**Comment by tw15egan** [comment]
Looks good to me! 👍 ✅

**Pull request merged**